gutils/gimage.c

```diff
@@ -416,3 +460,60 @@ Color GImageGetPixelColor(GImage *image,int x, int y) {
     struct _GImage *base = image->list_len==0?image->u.image:image->u.images[0];
 return( _GImageGetPixelColor(base,x,y));
 }
+
+const char* GImageHash(GImage *img) {
```

Comment:
So what is this used for

Reply:
Nothing, sorry. I deleted it. It was from an earlier stage of development.